image/mkosi/mkosi.files/mkosi.azure.conf

```diff
@@ -0,0 +1,3 @@
+[Output]
+KernelCommandLine=constel.csp=azure
```

Comment:
I don't know what mkosi does pass to the kernel by default, but I presume they don't automatically add `mitigations=auto,nosmt`as CoreOS does, right? If yes, should we add that for parity?